### DISCUSSION TITLE
Issues with benetel RAN550 b79 radio and srsRAN

### DISCUSSION THREAD

**Discussion by ptrauejr** [srsran/srsRAN_Project#963] [General Help]
We're trying to get an attach with a Cradlepoint R1900 on our system, but it doesn't seem to see the signal.  We know its not an issue with the CradlePoint as we have different n79 cells we can broadcast with and those work.  We're running 24.10 from github that we built.

We've got timesync via PTP:

![image](https://github.com/user-attachments/assets/1ff494c8-2b79-491d-93ff-6674104bfc9c)

SRS config file:

[benetel_550_tdd_n79_100mhz_4x2.yml.txt](https://github.com/user-attachments/files/17981959/benetel_550_tdd_n79_100mhz_4x2.yml.txt)

The benetel config file:

[ru_config.cfg.txt](https://github.com/user-attachments/files/17981977/ru_config.cfg.txt)

We're also seeing continuous messages from SRS saying:  "Missed incoming User-Plane PRACH messages for slot '0' and sector#0"

GNB Log file:

[gnb.log](https://github.com/user-attachments/files/17981990/gnb.log)

**Comment by BurmazDaniel** [comment]
Hi!

Any updates regarding this?
